docs/homeassistant/homeassistant-official.md

```diff
@@ -2,18 +2,34 @@
 
 To integrate your Z-Wave components you can use either the official Home Assistant [Z-Wave JS integration](https://www.home-assistant.io/integrations/zwave_js) and/or MQTT discovery.
 
-## Z-Wave JS server
+## Z-Wave JS integration
 
-The Z-Wave JS server is the official way to integrate your Z-Wave devices with Home Assistant. In order to use it go to Settings, [Home Assistant](/usage/setup?id=home-assistant) and enable the flag **WS Server**. Using this method, the official [Z-Wave JS integration](https://www.home-assistant.io/integrations/zwave_js) will automatically create entities in Home Assistant.
+Home Assistant has an integration for zwave which is based on zwave-js - you can read more about this at the [Z-Wave JS integration](https://www.home-assistant.io/integrations/zwave_js) docs.
```

Comment:
```suggestion
Home Assistant has an integration for Z-Wave which is based on Z-Wave JS. You can read more about this at the [Z-Wave integration](https://www.home-assistant.io/integrations/zwave_js) docs.
```

You removed the link to the HA settings which seems to still be useful since it describes the settings being set in these instructions.

Reply:
I would also make it very clear this is the reccomended way to discover entities and that MQTT is just an alternative (and not really maintained as I already say in `homeassistant-mqtt.md` docs)

Reply:
Would it make sense to move that here? 

I included it because this is the “home assistant” doc in the menu, and that the first paragraph said “two ways” and then proceeded to describe one way.

Reply:
I've removed the MQTT steps and linked to the page instead.